unimpeded/unimpeded.py

```diff
@@ -0,0 +1,29 @@
+#%%
+from anesthetic import read_chains
+#%%
+class unimpeded:
+
+    # which __init__ is better?
+    """
+    def __init__(self, **kwargs):
+        self.model = kwargs.pop('model', None)
+        self.data = kwargs.pop('data', None)
+        self.method = kwargs.pop('method', None)
+    """
+    def __init__(self, model, dataset, method, location):
+        self.model = model
+        self.dataset = dataset
+        self.method = method
```

Comment:
these should be arguments of `.get`. For now one would only want location as a `kwarg`